src/main/java/duke/Main.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+
+public class Main {
+    /**
+     * Main class to start the Duke chatbot.
```

Comment:
Should be an empty line between description and parameters.

Reply:
Thanks for spotting this, fixed in a later commit!